client/src/components/Sub_RatingsReviews/FormComponents/ReviewModal.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import { createPortal } from 'react-dom';
+import ReviewModalContent from './ReviewModalContent.jsx';
+
+export default function ReviewModal({ setShowModal, product, reviewMetadata}) {
+  const portal = document.getElementById('modal');
+  return createPortal(<div
+    className="review-comp-container"
+  >
+    <div
```

Comment:
Oh i see. this is where the magic happens! you can render a component as a child of a different DOM node than its parent component with the createPortal. learned. ty